querybook/webapp/redux/comment/reducer.ts

```diff
@@ -34,10 +34,9 @@ function commentReducer(state = initialState, action: CommentAction) {
             }
             case '@@comment/ARCHIVE_COMMENT': {
                 const { commentId } = action.payload;
-                draft.commentsById[commentId] = {
-                    ...draft.commentsById[commentId],
-                    archived: true,
-                };
+                draft.commentsById[commentId].archived = true;
+                draft.commentsById[commentId].text =
```

Comment:
nit: this line doesn't matter, I dont think we need to remove the text in user's redux yet